core/src/main/java/io/questdb/cairo/ColumnType.java

```diff
@@ -86,10 +86,12 @@ public final class ColumnType {
     public static final short GEOHASH = RECORD + 1;             // = 23
     public static final short LONG128 = GEOHASH + 1;            // = 24  Limited support, few tests only
     public static final short IPv4 = LONG128 + 1;               // = 25
+    public static final short VARCHAR = IPv4 + 1;               // = 26
```

Comment:
Needs a comment how `VARCHAR` is different from `STRING`. Pretty confusing as-is.
Possibly renaming to `UTF8` is the way to go?